### PR TITLE
Add complete README for logging

### PR DESCRIPTION
**Pull request by mziccard**

### CODE REVIEW

File: gcloud-java-logging/README.md [hunk 1]
```diff
@@ -15,32 +17,155 @@ Java idiomatic client for [Google Cloud Logging][cloud-logging].
...
+```java
+LoggingHandler.addHandler(LOGGER, new LoggingHandler());
+```
+After that, logs generated using `LOGGER` will be also directed to Stackdriver Logging.
+
+Notice that you can also register a `LoggingHandler` via the `logging.properties` configuration
+file. Adding, for instance, the following line:
+```
+com.google.cloud.examples.logging.snippets.AddLoggingHandler.handlers=com.google.cloud.logging.LoggingHandler
+```
```
Comment (lesv) [inline gcloud-java-logging/README.md]: Is there any reason we would choose one over the other?

File: gcloud-java-logging/README.md [hunk 2]
```diff
@@ -66,7 +193,9 @@ Contributing
 
 Contributions to this library are always welcome and highly encouraged.
 
-See [CONTRIBUTING] for more information on how to get started.
+See `gcloud-java`'s [CONTRIBUTING] documentation and the `gcloud-*` [shared documentation](https://github.com/GoogleCloudPlatform/gcloud-common/blob/master/contributing/readme.md#how-to-contribute-to-gcloud) for more information on how to get started.
```
Comment (lesv) [inline gcloud-java-logging/README.md]: `[CONTRIBUTING](../CONTRIBUTING.md)`

**Comment by lesv** [comment]
Just the one Question, if no, then Ok to merge.  (LGTM)

File: gcloud-java-logging/README.md (hunk 1 above)
Comment (mziccard) [inline gcloud-java-logging/README.md]: There could be depending on your application needs. But I would say that this does not depend on our handler but rather it's a design choice that comes with JUL.

Comment (lesv) [inline gcloud-java-logging/README.md]: LOL - it's reasonable to punt on this.  I agree.

**Pull request merged**